docs/docs/guides/evaluation/builtin_scorers.mdx

```diff
@@ -85,36 +80,31 @@ import TabItem from '@theme/TabItem';
     ```python
     from weave.scorers import SummarizationScorer
 
-    llm_client = ... # initialize your LLM client here
-
     scorer = SummarizationScorer(
-        client=llm_client,
-        model_id="gpt-4o"
+        model_id="openai/gpt-4o" # or any other model supported by litellm
     )
     ```
 
     **How It Works:**
 
     This scorer evaluates summaries in two ways:
 
-    1. **Entity Density:** Checks the ratio of unique entities (like names, places, or things) mentioned in the summary to the total word count in the summary in order to estimate the "information density" of the summary. Uses an LLM to extract the entities. Similar to how entity density is used in the Chain of Density paper, https://arxiv.org/abs/2309.04269
```

Comment:
Any reason why you removed the paper reference?

Reply:
good catch